src/layouts/components/Footer.tsx

```diff
@@ -59,12 +59,22 @@ function FooterExternalLink (props: { text: string, href: string }): JSX.Element
   )
 }
 
-function FooterInternalLink (props: { text: string, href: string }): JSX.Element {
+function FooterInternalLink (props: { text: string, pathname: string }): JSX.Element {
```

Comment:
I think we should standardize and use `href` instead.

Right now `FooterTinyLink` uses `href`, `FooterExternalLink` uses `url`, `FooterInternalLink` uses by they are more or less the same thing.

Reply:
renamed `href` to `url` for `FooterTinyLink` instead 

- url = external
- pathname = internal, it's kinda a limitation for the new `<Link/>` implementation so that I don't have to parse URL to URL to detect query string structure.